efficientdet/hparams_config.py

```diff
@@ -276,6 +276,9 @@ def default_detection_configs():
   # A temporary flag to switch between legacy and keras models.
   h.use_keras_model = True
 
+  # rectifies CPU memory leak
+  h.run_epoch_in_child_process = True
```

Comment:
I think you don't need this config, since you already have a flag in main.py?

Reply:
config is used to pass around different files, but since only main.py needs this, making it a FLAG in main.py is better.

Reply:
Thanks. I was also wondering why need to set the default in two places. Corrected now